lib/Doctrine/DBAL/Schema/AbstractAsset.php

```diff
@@ -46,10 +46,9 @@ protected function _setName(string $name) : void
             $this->_quoted = true;
             $name          = $this->trimQuotes($name);
         }
+        
```

Comment:
The extra spaces are what make the build fail. Also, you probably want to target a lower branch, I'm going to fix all this for you.

Reply:
I just saw this comment. Thanks again for your support. If there is something I can do, please let me know.